### ISSUE TITLE
Skip a cell in google sheet

### ISSUE THREAD

**Issue by Goretx** [DWilliames/google-sheets-sync-figma#154]
**TL;DR
It would be great to have the possibility to skip a cell in google sheet**

***

I've recently worked on a complex project where I had some sort of matrix of layouts:

stream 1: layout A1, layout B1, layout C1
stream 2: layout A2, layout B2, layout C2
stream 3: layout A3, layout B3, layout C3

while, for instance, all the C layouts where similar to each other, they didn't have the exact same number of elements. so C1 might have had 4 text fields, C2 only 2 text fields and C3 4 again.

I tryed to use Google sheets sync using a sheet's tab for all the As, a second tab for the Bs and a third tab for the Cs but having a different number of elements on the layouts messed up the process. I could have used the .(number-of-column) but there where so many cases of this thing happening that I ended up doing the entire process manually.

There might be ways to get around this problem or best practices that I haven't thought of, but having the possibility to skip a cell in google sheet could be the solution. maybe by using a special character.

thanks, love this plugin.
